app/styles/main.css

```diff
@@ -58,6 +58,87 @@ body {
   @apply !bg-white !text-corporate-blue;
 }
 
+.bk-panel-models-widgets-CustomSelect {
```

Comment:
@herrherrmann 

Could you maybe review the HTML / CSS / JS code in this PR? 🌝
I think you can do it faster and give more meaningful feedback than me

Reply:
I guess too late... :-/